IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -481,7 +481,7 @@ - (MSIDBaseToken *)getTokenByLegacyUserId:(NSString *)legacyUserId
             return nil;
         }
         
-        NSArray<MSIDBaseToken *> *matchedTokens = [MSIDTokenFilteringHelper filterRefreshTokenCacheItems:cacheItems legacyUserId:legacyUserId context:context];
+        NSArray<MSIDBaseToken *> *matchedTokens = [MSIDTokenFilteringHelper filterRefreshTokenCacheItems:cacheItems legacyUserId:legacyUserId environment:alias.msidHostWithPortIfNecessary context:context];
```

Comment:
nit: new lines would be good, this looks way too long on "some" screens :p

Reply:
done